packages/node_modules/@webex/plugin-meetings/src/common/constants.js

```diff
@@ -0,0 +1,6 @@
+// TODO: populate and test
```

Comment:
What's the reason for `common` rather than the other constants file?

Reply:
this is the external ones